asapdiscovery-modeling/asapdiscovery/modeling/workflows/protein_prep.py

```diff
@@ -249,6 +231,35 @@ def protein_prep_workflow(inputs: ProteinPrepInputs):
         target_kwargs={"target_name": "ref"},
         ligand_kwargs={"compound_name": "ref_ligand"},
     )
+
+    # setup the path to cached structures
+    cache_path = output_dir / inputs.gen_cache
+
+    # load the cache
+    try:
+        cached_complexs = ProteinPrepper.load_cache(cache_dir=cache_path)
```

Comment:
With load cache now outside the function other workflows now need to do same right? might be easier to hide behind the class.

Reply:
Fixed